react/components/EXPERIMENTAL_Table/Toolbar/InputSearch.tsx

```diff
@@ -12,7 +11,7 @@ const InputSearch: FC<InputSearchProps> = ({ onSubmit, ...inputProps }) => {
   return (
     <form
       id={NAMESPACES.TOOLBAR.INPUT_SEARCH}
-      className={csx(ORDER_CLASSNAMES.TOOLBAR_CHILD.INPUT, 'w-40')}
+      className={ORDER_CLASSNAMES.TOOLBAR_CHILD.INPUT}
```

Comment:
```suggestion
      className={csx(ORDER_CLASSNAMES.TOOLBAR_CHILD.INPUT, 'w-40-ns')
```
maybe ?

Reply:
You could also add `w-100-s` to make it more appealing.

Reply:
It works well only if put `flex-wrap`only in small screens, because of this:

![Screenshot from 2020-01-10 14-04-17](https://user-images.githubusercontent.com/6867958/72172074-07b9c880-33b3-11ea-822d-27575ceead6b.png)

But I'm not so sure if it's good to wrap only on small screens.

Reply:
Oh, I see. I think is up to the design team to decide if it's okay.